### PR TITLE
TCA-766 focus after navigate next item

### PR DESCRIPTION
**Pull request by lecosson**
Related to : https://oat-sa.atlassian.net/browse/TCA-766
 
Move focus to question on navigate to item
  
#### How to test

- prepare an instance of ACT
- execute test as a test taker with enabled accesibility mode
- check focus is moved to question on navigate to next item
 
### related PR

https://github.com/oat-sa/extension-tao-testqti/pull/1918

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-test-runner-qti",
-    "version": "2.16.3",
+    "version": "2.16.4",
```
Comment (atsymuk) [inline package.json:3]: version 2.16.4 has already been taken

**Comment by atsymuk** [comment]
it is not allowed to create PRs into master branch

**Review by atsymuk — APPROVED** [review]
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
Code review only
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**